src/ring_theory/witt_vector/domain.lean

```diff
@@ -0,0 +1,209 @@
+/-
+Copyright (c) 2022 Robert Y. Lewis. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Robert Y. Lewis
+-/
+
+import ring_theory.witt_vector.identities
+
+/-!
+
+# Witt vectors over a domain
+
+This file builds to the proof `witt_vector.domain`,
+an instance that says if `R` is an integral domain, then so is `𝕎 R`.
+Most of the file develops an API around iterated applications
+of `witt_vector.verschiebung` and `witt_vector.frobenius`.
+
+The [proof sketch](https://tinyurl.com/2p8cwrn7) goes as follows:
```

Comment:
I'd personally prefer a link that makes it clear what its linking to, and I don't really see a need to minify the url, the long-lines check already special-cases lines with a URL in